### PR TITLE
Add SC16IS750/760xPW and SC16IS750/760xBS

### PR DESCRIPTION
**Pull request by kacper-ka**
This merge adds 4 new symbols:
* **SC16IS750xPW**,
* *SC16IS760xPW* as an alias for **SC16IS750xPW**,
* **SC16IS750xBS**,
* *SC16IS760xBS* as an alias for **SC16IS750xBS**.

These ICs are very similar to **SC16IS740** already present in the library, however they are equipped with additional GPIO lines and are available in different packages: TSSOP-24 and HVQFN-24.

#### Datasheet
https://www.nxp.com/docs/en/data-sheet/SC16IS740_750_760.pdf

#### Related PRs
pointhi/kicad-footprint-generator#486
KiCad/kicad-footprints#2060

#### Screenshots
SC16IS750xBS
SC16IS750xPW

### KLC violations
None.
---
- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
- [x] A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

**Comment by kacper-ka** [comment]
@chschlue the `kicad-foorprints` library does not contain 24-pin VQFN footprint with 2.5x2.5mm EP, do you suggest I should add it as well? NXP specifies the EP size and its nominal dimensions are 2.45..2.75x2.45..2.75mm - don't you think that using a 2.45x2.45mm EP is acceptable?

**Comment by chschlue** [comment]
NXP does not specify nominal dimensions, just lower and upper bounds, but [suggests](https://www.nxp.com/docs/en/package-information/SOT616-3.pdf) 2.5x2.5mm, so that should be used as per [F6.3](https://kicad-pcb.org/libraries/klc/F6.3/).

Adding that footprint isn't hard, you basically just add the dimensions to https://github.com/pointhi/kicad-footprint-generator/blob/master/scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml and open a PR.

File: Interface_UART.dcm [hunk 1]
```diff
@@ -375,6 +375,30 @@ K UART I2C/SPI 64B FIFO IrDA SIR
...
+$ENDCMP
+#
+$CMP SC16IS750xPW
+D Single UART with I2C/SPI interface, 64 bytes of transmit and receive FIFOs, IrDA SIR built-in support, TSSOP-24
+K UART I2C/SPI 64B FIFO IrDA SIR
+F https://www.nxp.com/docs/en/data-sheet/SC16IS740_750_760.pdf
+$ENDCMP
+#
+$CMP SC16IS760xBS
+D Single UART with I2C/SPI interface, 64 bytes of transmit and receive FIFOs, IrDA SIR built-in support, VQFN-24
```
Comment (chschlue) [inline Interface_UART.dcm:391]: Please change package here, too.

Comment (kacper-ka) [reply Interface_UART.dcm:601]: Right, missed it.

**Pull request merged**